samples/cxx/flamespeed/flamespeed.cpp

```diff
@@ -141,18 +137,16 @@ int flamespeed(double phi, bool refine_grid, int loglevel)
         flame.solve(loglevel,refine_grid);
         double flameSpeed_mix = flame.value(flowdomain,
                                             flow->componentIndex("velocity"),0);
-        print("Flame speed with mixture-averaged transport: {} m/s\n",
-              flameSpeed_mix);
-        flame.save(fileName, "mix", "Solution with mixture-averaged transport", 0);
+        print("Flame speed with mixture-averaged transport: {} m/s\n", flameSpeed_mix);
```

Comment:
Does this actually fill in the variable to the string? I see it was this way before, but is this a bug to fix?

Reply:
Output relies on `fmt::print` and works as expected. For `phi=1.`, the output is
```
Flame speed with mixture-averaged transport: 0.3800463682237259 m/s
```

Reply:
Oh, thanks. I didn't see this was cpp code! 🤪